stubs/resource_collection.blade.php

```diff
@@ -0,0 +1,14 @@
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\ResourceCollection;
+use Illuminate\Support\Arr;
+
+class {{$entity}}ResourceCollection extends ResourceCollection
+{
+    public $collects = {{$entity}}Resource::class;
+
+    public function paginationInformation($request, $paginated, $default)
+    {
+        return Arr::except($paginated, 'data');
+    }
```

Comment:
let's use new pagination style for all new projects. I've discussed it with the RN team

```suggestion
```